service/OrderModel.go

```diff
@@ -126,3 +126,14 @@ func (s *OrderService) AssigningOrderToCourier(order dao.Order) error {
 	}
 	return nil
 }
+
+func (s *OrderService) GetDetailedOrdersById(Id int) (dao.DetailedOrder, error) {
+	var Order dao.DetailedOrder
+	Order, err := s.repo.GetDetailedOrdersByIdFromDB(Id)
+	if err != nil {
+		err := errors.New("no id")
+		log.Println("no more id")
```

Comment:
Have you meant not found? because no id sounded like you haven't passed an id to the function

Reply:
Also if you start returning all other errors from the repo, you won't be able to rely anymore on the fact that every error means not found, it could mean anything else like couldn't start transaction.